Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Platform/Windows/platform_windows.cmake

```diff
@@ -6,3 +6,5 @@
 #
 #
 
+# Temporary fix for the linker issue caused by OpenEXR libs on debug builds
+set(LY_LINK_OPTIONS INTERFACE "$<$<CONFIG:debug>:/INCREMENTAL>")
```

Comment:
This linkere option should be placed in the compiler specific cmake settings file located in the [Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Platform/Common](https://github.com/o3de/o3de/tree/abb635e583436d72d5e9a9b712dbb7b448cc94e6/Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Platform/Common) directory.

Reply:
oh awesome that would make more sense.